client/blocks/props-viewer/index.jsx

```diff
@@ -0,0 +1,208 @@
+/**
+ * External Dependencies
+ */
+import React, { PropTypes, PureComponent } from 'react';
+
+/**
+ * Internal Dependencies
+ */
+import Card from 'components/card';
+import Gridicon from 'components/gridicon';
+
+// todo: move this file!
+const components = require( '../../../server/devdocs/proptypes-index.json' );
```

Comment:
>I don't think so... I tried that and it blew up

How did it blow up? Using `import` on my local branch it seems to work fine for me.

Also, is `server` not part of `NODE_PATH` in this case? If it were, we could simply import from `devdocs/proptypes-index.json`.